config/webhook.yaml

```diff
@@ -1,32 +1,38 @@
 #@ load("@ytt:data", "data")
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
   name: defaults.webhook.kpack.io
 webhooks:
 - admissionReviewVersions:
+  - v1
   - v1beta1
   clientConfig:
     service:
       name: kpack-webhook
       namespace: kpack
   failurePolicy: Fail
+  matchPolicy: Exact
   sideEffects: None
+  timeoutSeconds: 30
```

Comment:
Seems like if the new default is `10` and other projects are going with that: https://github.com/tektoncd/pipeline/blob/main/config/500-webhooks.yaml we should maybe just take the default?